inlanecruising_plugin/include/inlanecruising_plugin/inlanecruising_plugin.h

```diff
@@ -150,6 +150,32 @@ class InLaneCruisingPlugin
                                                   double max_starting_downtrack,
                                                   const carma_wm::WorldModelConstPtr& wm);
 
+  /**
```

Comment:
All the changes in this folder seem unrelated to PR feature set. This seems like an invalid merge